Makefile

```diff
@@ -45,3 +45,8 @@ test-chapter4:
 test-chapter4-basic:
 	cabal test doctest-chapter4 --enable-tests --test-show-details=direct
 	cabal run learn4haskell-test --enable-tests -- -m "Chapter4Normal"
+
+.PHONY: test-all
+test-all:
+	cabal test all --enable-tests --test-show-details=direct
+	cabal run learn4haskell-test --enable-tests --
```

Comment:
I think this line is redundant. `cabal test all` should already run all tests, including `doctest`s :slightly_smiling_face:

Reply:
Fixed